static/app/views/replays/detail/layout/asideTabs_v2.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from 'react';
+import styled from '@emotion/styled';
+
+import NavTabs from 'sentry/components/navTabs';
+import Placeholder from 'sentry/components/placeholder';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import {t} from 'sentry/locale';
+import ReplayReader from 'sentry/utils/replays/replayReader';
+
+import TagPanel from '../tagPanel';
+
+import ResizePanel from './resizePanel';
+import {BreadCrumbsContainer, VideoContainer} from '.';
```

Comment:
This is creating a circular import which we should avoid.
```
index.tsx -> imports asideTabs_v2.tsx -> imports index.tsx
```

If we move these two components into here then index.tsx would import them and break the cycle. But it looks like they depend on the shared `<PageColumn>` component. So instead I'd recommend making a new file that all the `*Section` styled components can live in, then both index.tsx and asideTabs_v2.tsx would import what they need.